### PR TITLE
DM-35487: Tweak cardinal directions for All Sky camera

### PR DESCRIPTION
**Pull request by ugyballoons**

### CODE REVIEW

File: src/rubintv/handlers/external/endpoints.py [hunk 1]
```diff
@@ -334,6 +349,10 @@ def month_names() -> List[str]:
     return [date(2000, m, 1).strftime("%B") for m in list(range(1, 13))]
 
 
+def get_static_resource_timestamp() -> int:
+    return int(os.path.getmtime("src/rubintv/static/stylesheets/main.css"))
```
Comment (jonathansick) [inline src/rubintv/handlers/external/endpoints.py:353]: This'll work, but I think there's a limitation here. I don't think [git tracks filesystem metadata like modification date](https://stackoverflow.com/a/65550038) so this timestamp effectively gets set when the Docker image gets built (my expectation, at least). So this'll successfully burst caches, but won't let a browser used the cached style sheet or other asset files between rubintv builds — which isn't much of a performance loss.

The way I'd kind of expect to implement this is by [doing some sort of hash calculation](https://docs.python.org/3/library/hashlib.html) on the content during app start up and cache that in memory (like in the aiohttp app context or via `functools.cache`). But again, no real harm in what you've got.

**Review by jonathansick — APPROVED** [review]
Looks good. I've just got a recommendation to cache the assert versioning function.

File: src/rubintv/handlers/external/endpoints.py (hunk 1 above)
Comment (ugyballoons) [reply src/rubintv/handlers/external/endpoints.py:353]: Yes, a hash generator that ran on startup across all the static files would be much better. I think that would take me a while to write though.

It's frustrating that aiohttp offers just this functionality but I can't find anything in the docs about how to use the corresponding `url_for()` function in the templates.

File: src/rubintv/handlers/external/endpoints.py [hunk 2]
```diff
@@ -334,6 +349,10 @@ def month_names() -> List[str]:
     return [date(2000, m, 1).strftime("%B") for m in list(range(1, 13))]
 
 
+def get_static_resource_timestamp() -> int:
```
[the remark this answers is not in the document]
Comment (jonathansick) [reply src/rubintv/handlers/external/endpoints.py:352]: Right. It's a function decorator, so it'll look like:

```suggestion
import functools

@functools.cache
def get_static_resource_timestamp() -> int:
```

**Pull request merged**